viewslides.py

```diff
@@ -148,7 +149,7 @@ def save(self):
         pickle_output.close()
 
 
-class ReadHTTPRequestHandler(network.ChunkedGlibHTTPRequestHandler):
+class ReadHTTPRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
```

Comment:
Did you get any errors while trying to use `sugar3.network`?

Reply:
I didn't try that! I'll try using that instead!